superset-frontend/src/explore/components/AdhocMetricEditPopover.jsx

```diff
@@ -50,7 +50,11 @@ const defaultProps = {
   columns: [],
 };
 
-const startingWidth = 300;
+const ResizeIcon = styled.i`
```

Comment:
I see this is happening a couple times in this PR, but I can't think of an easy way to DRY this up that would be worth doing if we come through and start replacing all the font-awesome icons in another pass anyway. Only mentioning it in case you have an idea of how to simply consolidate the two `ResizeIcon` instances... but it's probably not worth worrying about. So now I'm just rambling...

Reply:
I felt bad when repeating that code, but I couldn't see an easy way not to 🙁 In general, I think it might be worth it to refactor Metric and Filter popovers, because they are rather similar, with only tabs content differing.